  0%|                                                                            | 0/500 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/data1/lihaobo/tracking/rft/src/scripts/run_tracking_debug.py", line 43, in <module>
    main2()
  File "/data1/lihaobo/tracking/rft/src/virft/src/open_r1/grpo_tracking.py", line 509, in main2
    main(script_args, training_args, model_args)
  File "/data1/lihaobo/tracking/rft/src/virft/src/open_r1/grpo_tracking.py", line 500, in main
    trainer.train()
  File "/home/lihaobo/miniconda3/envs/track/lib/python3.10/site-packages/transformers/trainer.py", line 2243, in train
    return inner_training_loop(
  File "/home/lihaobo/miniconda3/envs/track/lib/python3.10/site-packages/transformers/trainer.py", line 2550, in _inner_training_loop
    tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
  File "/home/lihaobo/miniconda3/envs/track/lib/python3.10/site-packages/transformers/trainer.py", line 3694, in training_step
    inputs = self._prepare_inputs(inputs)
  File "/data1/lihaobo/tracking/rft/src/virft/src/open_r1/trainer/grpo_trainer_tracking.py", line 311, in _prepare_inputs
    for k, v in inputs.items():
AttributeError: 'list' object has no attribute 'items'
[rank0]: Traceback (most recent call last):
[rank0]:   File "/data1/lihaobo/tracking/rft/src/scripts/run_tracking_debug.py", line 43, in <module>
[rank0]:     main2()
[rank0]:   File "/data1/lihaobo/tracking/rft/src/virft/src/open_r1/grpo_tracking.py", line 509, in main2
[rank0]:     main(script_args, training_args, model_args)
[rank0]:   File "/data1/lihaobo/tracking/rft/src/virft/src/open_r1/grpo_tracking.py", line 500, in main
[rank0]:     trainer.train()
[rank0]:   File "/home/lihaobo/miniconda3/envs/track/lib/python3.10/site-packages/transformers/trainer.py", line 2243, in train
[rank0]:     return inner_training_loop(
[rank0]:   File "/home/lihaobo/miniconda3/envs/track/lib/python3.10/site-packages/transformers/trainer.py", line 2550, in _inner_training_loop
[rank0]:     tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
[rank0]:   File "/home/lihaobo/miniconda3/envs/track/lib/python3.10/site-packages/transformers/trainer.py", line 3694, in training_step
[rank0]:     inputs = self._prepare_inputs(inputs)
[rank0]:   File "/data1/lihaobo/tracking/rft/src/virft/src/open_r1/trainer/grpo_trainer_tracking.py", line 311, in _prepare_inputs
[rank0]:     for k, v in inputs.items():
[rank0]: AttributeError: 'list' object has no attribute 'items'
